README.md

```diff
@@ -49,6 +49,12 @@ Default: "vjs-playlist"
 
 The name of the class to search for to populate the playlist menu.
 
+#### playOnClick
```

Comment:
Naming nitpick: click is pretty mouse-specific. What about `playOnSelect` to account for touch interactions?